src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -38,12 +38,18 @@ def create_rss_notifications(event)
 
   def notification_dynamic_params(event)
     case event.eventtype
-    when 'ReviewWanted', 'RequestCreate', 'RequestStatechange'
+    when 'RequestStatechange'
       {
         notifiable_type: 'BsRequest',
         notification_id: event.payload['id'],
-        bs_request_oldstate: (event.payload['oldstate'] if event.eventtype == 'RequestStatechange')
-      }.compact
+        bs_request_state: event.payload['state'],
+        bs_request_oldstate: event.payload['oldstate']
+      }
+    when 'ReviewWanted', 'RequestCreate'
+      {
+        notifiable_type: 'BsRequest',
```

Comment:
:thought_balloon:  Can't you just use a guard clause inside the `when`?

Reply:
Sure. On it.

Reply:
Done. Please re-review.